styles/main.css

```diff
@@ -17,5 +20,5 @@ nav ul li a {
 	color:#ffaa00;
 }
 .byline{
-	color:#ffcc00;
+	color:rgba(255, 204, 0, 0.5);
```

Comment:
The lab asked for a transparent color.  Here I use the rgba  notation of the same colour (`#ffcc00` = `rgb(255,  204, 0)`) and give it an alpha  value of 0.5, which makes it 50% transparent